webapp/src/shared/components/RowsList/RowsList.tsx

```diff
@@ -0,0 +1,14 @@
+import './RowsList.css';
+import Row, { RowProps } from '../Row/Row';
+
+export type RowsListProps = {
+  rows?: RowProps[];
+};
+
+export default function RowsList({ rows }: RowsListProps) {
+  return (
+    <div className="rows-list-wrapper">
+      {rows && rows.map((rowProps) => <Row {...rowProps} />)}
+    </div>
```

Comment:
suggestion: Here we could use the unordered list and list items tags to improve the semantics.
Also, when mapping over an array, [React docs](https://reactjs.org/docs/lists-and-keys.html#keys) recommends adding a `key` property to the items.

Something like this:
```typescript
   <ul className="rows-list">
      {rows &&
        rows.map((rowProps) => (
          <li className="rows-list-item" key={rowProps.title}>
            <Row {...rowProps} />
          </li>
        ))}
    </ul>
```

```css
.rows-list {
  width: 100%;
  list-style: none;
  margin: 0;
  padding: 0;
}

.rows-list-item:not(:last-child) {
  margin-bottom: 1.75rem;
}
```

Reply:
better now?

Reply:
Maybe `title` isn't stable enough as a key, since we may have duplicated titles, I think.

Reply:
You are right :100:
Also, the `title` is optional, and it could be `undefined`.
Any ideas on what we could use as a `key`?

Reply:
maybe we should add a id prop to Row, and use the uuid of the user?

Reply:
Sure, using the user UUID seems like a good option to me. Maybe we could name the prop `key` because `id` has a special purpose in HTML.
Or we could also use `id` without passing it as a prop to the `button` tag inside the Row component.

Reply:
good catch! I like the idea of adding an addtional `id` prop to the row component :+1:

Reply:
A key in the row component is a good idea and using user UUID seems good as well

Reply:
Case in point for the :not(:lastchild:) pseudo selector here https://dev.to/puputtallulah/stop-using-ids-and-classes-in-every-html-element-3k2c

Reply:
Do we like where the key is [now](https://github.com/42AGV/ft_transcendence/pull/102/commits/37b6d66f8b2f3c95a1ddb736af586e0657429517)?

Reply:
I like the idea of adding the `key` only to the RowListProps :+1:
We could improve the naming of the types. What do you think about `RowItem` and `rowProps`?

RowEntity -> RowItem
rowDto -> rowProps

Another option is to create a type that extends the RowProps type:

```
type RowItem = RowProps & {
  key: string;
};
```

Then we could use destructuring while mapping the rows:
```
        rows.map((rowListProps) => {
          const { key, ...rowProps } = rowListProps;
          return (
            <li className="rows-list-item" key={key}>
              <Row {...rowProps} />
            </li>
          );
        }
```

Reply:
Now?

Reply:
@Damrod, thank you. LGTM :rocket: